src/MyReactComponent.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class MyReactComponent extends React.Component {
+    render() {
+        return <div>Hello from React component</div>
+    }
+}
+
+export default MyReactComponent
```

Comment:
don't forget about empty line at the end of the files

Reply:
Ohh, yes. Will fix it!